src/apis/timestamp_api.rs

```diff
@@ -4,16 +4,22 @@
  * Rekor is a cryptographically secure, immutable transparency log for signed software releases.
  *
  * The version of the OpenAPI document: 0.0.1
- * 
+ *
  * Generated by: https://openapi-generator.tech
  */
 
-
+use super::{configuration, Error};
+use crate::apis::ResponseContent;
 use reqwest;
 
-use crate::apis::ResponseContent;
-use super::{Error, configuration};
+// Case 1:
+use std::fs;
+use std::io::Write;
+
+use std::os::unix::fs::PermissionsExt;
+use std::path::PathBuf;
 
+const FILENAME: &str = "response.tsr";
```

Comment:
Do we really need to add this binary blob to the repository? Can't we let the use provide it or have the code fetch it somehow based on user input?

If the binary file must be part of the repo, at least move it to a dedicated folder (dunno `/assets`) instead of having it inside of the root of the repository

Reply:
The timestamp response is saved in the file response.tsr when the user makes `get_timestamp_response` api call.
The user is supposed to provide timestamp request file (an example for the input file is in https://github.com/jyotsna-penumaka/rekor-rs/blob/rekor-functionality/test_data/test_request.tsq )
This file doesn't need to be part of the repo, it is generated each time the user calls `get_timestamp_response` api call.
I can move the place where it is stored to be inside test_data.